rootdir/vendor/etc/seccomp_policy/atfwd@2.0.policy

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2020 Qualcomm Technologies, Inc.
+# All Rights Reserved.
```

Comment:
please remove this patch

Reply:
done

Reply:
please remove the patch !

Reply:
I have removed the patch, can you check https://github.com/sonyxperiadev/device-sony-common/pull/903/commits again?